Source/Diagnostics/ReducedDiags/LoadBalanceCosts.cpp

```diff
@@ -50,8 +50,8 @@ void LoadBalanceCosts::ComputeDiags (int step)
     const size_t dataSize =
         static_cast<size_t>(m_nDataFields)*
         static_cast<size_t>(nBoxes);
-    m_data.resize(dataSize, 0.0);
-    m_data.assign(dataSize, 0.0);
+    m_data.resize(dataSize, amrex::Real(0.0));
+    m_data.assign(dataSize, amrex::Real(0.0));
```

Comment:
You should be able to do this, if you want to be more compact
```suggestion
    m_data.resize(dataSize, 0.0_rt);
    m_data.assign(dataSize, 0.0_rt);
```